pydm/widgets/archiver_time_plot.py

```diff
@@ -65,20 +65,28 @@ def to_dict(self) -> OrderedDict:
         dic_.update(super(ArchivePlotCurveItem, self).to_dict())
         return dic_
 
-    def setArchiveChannel(self, address: str) -> None:
+    @TimePlotCurveItem.address.setter
```

Comment:
im wondering why '@TimePlotCurveItem.address.setter' instead of a '@address.setter' for the current class?

Reply:
This was a way for me to reference the super class' property without overriding the getter.

Looking back through the code, this only saves a few lines of code while being harder to read. Instead we could consider this if you think it is easier to read:
```suggestion
    @property
    def address(self):
        return super().address
        
    @address.setter
    def address(self, new_address: str) -> None:
```

Reply:
i like this, find it easier to read